### DISCUSSION TITLE
Incremental migration of changed pages in confluence after initial run

### DISCUSSION THREAD

**Discussion by GaryComp** [WikiTransformationProject/wikitraccs-releases#125] [General]
Hi, We used the licenced version for the migration, and unfortunately, access to Confluence wasn't disabled quickly enough. So my question would be, if the tool is rerun, will it pick up any changes with the pages on the Confluence side and migrate them to SharePoint again to capture the updates?
Or is there another way to do this?

**Comment by heinrich-ulbricht** [comment]
Hi @GaryComp, WikiTraccs does not overwrite existing content by default. But you can tell it explicitly which SharePoint pages to overwrite.

Here are the details: https://www.wikitransformationproject.com/docs/reference/update-old-pages/
